### DISCUSSION TITLE
Click or Tap with 3 Fingers is unavailable

### DISCUSSION THREAD

**Discussion by BenArviv** [artginzburg/MiddleClick#86] [Q&A]
Dear developer,

I really enjoy the app, but I noticed that that function isn't working. I've downloaded it via Homebrew, and the option for "Tap to click" works perfectly, but I prefer click instead of tap. Is there a way to fix that issue? 

Thank you very much,
Ben

**Comment by snowywol** [comment]
This question seems to be answered by issue  https://github.com/artginzburg/MiddleClick-Sonoma/issues/94 which is now closed.
